lldb/source/Plugins/SymbolFile/DWARF/DWARFASTParserSwiftDescriptorFinder.cpp

```diff
@@ -32,6 +32,29 @@ using namespace lldb_private;
 using namespace lldb_private::dwarf;
 using namespace lldb_private::plugin::dwarf;
 
+/// Given a die to a substituted generic Swift type, return the analogous
+/// unsubstituted version.
+///
+/// Given a generic type (for example, a generic Pair), the compiler will emit
+/// full debug information for the unsubstituted type (Pair<T, U>), and opaque
+/// debug information for each every specialization (for example, Pair<Int,
+/// Double>), with a link back to the unsubstituted type. When looking up one of
+/// the specialized generics, return the unsubstituted version instead.
+static std::optional<std::pair<CompilerType, DWARFDIE>>
+findUnsubstitutedGenericTypeAndDie(TypeSystemSwiftTypeRef &ts,
```

Comment:
Is `Die` the conventional way of casing in a function name? Or should it be `DIE`?